react/ProductList.tsx

```diff
@@ -85,39 +83,12 @@ const ItemContextWrapper = memo<ItemWrapperProps>(function ItemContextWrapper({
     ]
   )
 
-  return <ItemContextProvider value={context}>{children}</ItemContextProvider>
-})
-
-const ProductGroup: React.FC<Props> = (props) => {
-  const { items, renderOnView, userType, allowManualPrice, children } = props
-  const shouldAllowManualPrice =
-    allowManualPrice && userType === CALL_CENTER_OPERATOR
-
-  const { hasBeenViewed, dummyElement } = useRenderOnView({
-    lazyRender: true,
-    offset: 900,
-  })
-
-  if (renderOnView && (!hasBeenViewed || items.length === 0)) {
-    return dummyElement
-  }
-
   return (
-    <>
-      {items.map((item: ItemWithIndex) => (
-        <ItemContextWrapper
-          key={`${item.uniqueId}-${item.sellingPrice}-${item.index}`}
-          item={item}
-          itemIndex={item.index}
-          shouldAllowManualPrice={shouldAllowManualPrice}
-          {...props}
-        >
-          {children}
-        </ItemContextWrapper>
-      ))}
-    </>
+    <LazyRender height={100} offset={300}>
```

Comment:
any chance we can make this into a prop so agencies can customize the product height?

Reply:
I thinks it's a good suggestion. I'll review it.